src/database/migrations/20191118163016-create-users.js

```diff
@@ -26,6 +26,9 @@ module.exports = {
       signupType: {
         type: Sequelize.STRING
       },
+      role: {
+        type: Sequelize.STRING
+      },
```

Comment:
I have named it to `roleId`  and assign default value to be 3.
the column structure 
``` js
roleId : {
        type: Sequelize.INTEGER,
        defaultValue: 3,
        references: {  
          model: 'roles',
          key: 'id'
        }
      },
```